sorts.py

```diff
@@ -68,10 +68,33 @@ def chris_insert_sort(values):
 
 
 def peter_insert_sort(values):
-    # TODO: Test for empty input.
-    # TODO: Build sorted linked list from input using Link class.
-    # TODO: Copy sorted linked list into Python list.
-    pass
+    newlist = []
+    if len(values) <= 1:
+        return values
+    else:
+        head = Link(values[0], None)
+        current = head
+        for num in values[1:]:
+            modified = False
+            if num < head.value:
+                head = Link(num, head)
+                current = head
+            else:
+                while not modified:
```

Comment:
Ok so I know I just said to move "modified = False" just above this but I don't think you need modified it at all. If you change this to "while True:" and change the "modified = True" lines below to "break" it accomplishes the same thing.